site-src/v1alpha2/references/policy-attachment.md

```diff
@@ -0,0 +1,294 @@
+# Policy Attachment
+
+Policies attached to Gateway API resources and implementations should use the
+following approach to ensure consistency across implementations of the API.
+There are three primary components of this pattern:
+
+* A shared means of attaching policy to resources.
+* Support for configuring both default and override values within policy
+  resources.
+* A shared hierarchy to illustrate how default and override values should
+  interact.
+
+This kind of standardization will not only enable consistent patterns, it will
+enable tooling such as kubectl plugins to be able to visualize all policies that
+have been applied to a given resource.
```

Comment:
Nit: We're already doing this, so "will enable" is not really correct.
```suggestion
This kind of standardization not only enables consistent patterns, it allows
future tooling such as kubectl plugins to visualize all policies that
have been applied to a given resource.
```

Reply:
Updated, thanks!